test/data/activeLduClientTest.js

```diff
@@ -0,0 +1,52 @@
+const proxyquire = require('proxyquire')
+
+proxyquire.noCallThru()
+
+describe('Save and retrive LDU codes', () => {
+  let queryStub
+
+  const lduCode = 'ABCDE'
+
+  const lduProxy = (query = queryStub) => {
+    return proxyquire('../../server/data/activeLduClient', {
+      './dataAccess/db': {
+        query,
+      },
+    })
+  }
+
+  describe('doesLduExist', () => {
+    it('should call query once', () => {
+      queryStub = sinon.stub().resolves()
+      lduProxy().doesLduExist(lduCode)
+      expect(queryStub).to.have.callCount(1)
```

Comment:
these tests aren't really testing anything, would be worth looking at some of the other examples.

e.g, ones which check the statement and provided args: 
```
    const expectedParameters = ['NEW_STAGE', 'ABC123']

      const result = licencesProxy().updateStage('ABC123', 'NEW_STAGE')

      return result.then(() => {
        const { values } = queryStub.getCalls()[0].args[0]
        expect(values).to.eql(expectedParameters)
      })
    })
```

Reply:
or ```const expectedClause =
        'insert into licences (booking_id, licence, stage, version, vary_version) values ($1, $2, $3, $4, $5)'

      const result = licencesProxy().createLicence('ABC123')

      return result.then(() => {
        expect(queryStub.getCalls()[0].args[0].text).includes(expectedClause)
      })```

(there are better examples in use-of-force)

Reply:
have revised tests using test/data/licenceClientTest.js as examples